util/testslide-snippets/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Change Log
+## 1.0.0
```

Comment:
@david-caro  do you think it makes sense to sync this with TestSlide's version? If yes, we should also update `RELEASE.md` to cover this.

Reply:
Done